interface/modules/custom_modules/oe-module-weno/src/Services/TransmitProperties.php

```diff
@@ -379,10 +383,7 @@ private function getPatientInfo(): mixed
      */
     public static function styleErrors($error): string
     {
-        $log = "<div><p style='font-size: 1.0rem; color: red;'>" .
-            $error . "<br />" . xlt('Please address errors and try again!') .
-            "</p></div>";
-        return $log;
+        return "<div><p style='font-size: 1.0rem; color: red;'>" . $error . "</p></div>";
```

Comment:
does $error above need to be html escaped?

Reply:
yep having trouble multitasking these days so you may catch me a lot!